lib/_http_outgoing.js

```diff
@@ -354,6 +372,17 @@ function _writeRaw(data, encoding, callback) {
     encoding = null;
   }
 
+  let byteLen;
+  if (typeof data === 'string') {
+    byteLen = Buffer.byteLength(data, encoding);
+  } else {
+    byteLen = data.length;
+  }
```

Comment:
```suggestion
  const byteLen = Buffer.byteLength(data, encoding);
```

Reply:
@mcollina I believe you had some performance concerns in the past regarding this?

Reply:
We'd have to benchmark which one is best.